subprojects/language-groovy/src/main/java/org/gradle/api/internal/tasks/compile/NormalizingGroovyCompiler.java

```diff
@@ -86,22 +74,6 @@ public boolean apply(File element) {
         spec.setSourceFiles(ImmutableSet.copyOf(filtered));
     }
 
-    private void resolveClasspath(GroovyJavaJointCompileSpec spec) {
```

Comment:
So nothing fails after removing this? I'd expect some tests to fail when Groovy uses Java files in mixed & joint compilation. Do we have some tests for this?

Reply:
Hmm... Maybe this was necessary when the Groovy and Java output still went into the same directories. Do you have tests for Joint compilation with a forking Java compiler as well? I suppose Groovy would need to generate stubs there, then compile the Java code against those stubs and finally compile the Groovy code together with the Java code. Though I suppose those tests should already exist.

Reply:
I don't think there are any for forking compiler. I can add them